packages/@lightningjs/ui-components/src/components/Tooltip/Tooltip.stories.js

```diff
@@ -58,24 +58,41 @@ export default {
   }
 };
 
-export const Basic = () =>
+const sharedArgActions = {
+  argActions: {
+    delayVisible: (delayVisible, component) => {
+      component.tag('Tooltip').delayVisible = delayVisible;
+    },
+    timeVisible: (timeVisible, component) => {
+      component.tag('Tooltip').timeVisible = timeVisible;
+    }
+  }
+};
+
+export const Basic = args =>
   class Basic extends lng.Component {
     static _template() {
       return {
         Tooltip: {
           type: TooltipComponent,
-          x: 1280 / 2
+          x: 1280 / 2,
+          delayVisible: args.delayVisible,
+          timeVisible: args.timeVisible
         }
       };
     }
   };
 
+Basic.parameters = { ...sharedArgActions };
+
 export const LongTitle = Basic.bind({});
 LongTitle.args = {
   title:
     'This is a long message. Text will remain on a single line and does not have a maximum width'
 };
 
+LongTitle.parameters = { ...sharedArgActions };
```

Comment:
Same as above, can we just assign directly instead? Or do we want to leave it like this to make it clear that other properties can be added? I'm fine either way.

Reply:
Yeah, as you said I use spread operator for that reason but I checked other components for consistency some of them use spread and some don't. I think either way is fine.